### DISCUSSION TITLE
Improve options for recording physical locations of specimens

### DISCUSSION THREAD

**Discussion by ljwalker** [Symbiota/community-board#387] [Wish List]
Ecdysis user requests improved functionality for recording the physical location of a specimen within drawers and other sub containers (e.g. trays). They are currently using the _Alternative Identifiers_ table with a key-value pair as a placeholder method.

**Comment by themerekat** [comment]
This _should_ be stored in the Darwin Core field "disposition", though admittedly, this is just a single text field at the moment.

**Comment by ljwalker** [comment]
For collections that wish to record storage at a more granular level (cabinet #, freezer #, drawer #, rack #, tray #, vial #, etc.), the DwC recommended controlled vocabulary for _disposition_ is insufficient ("in collection", "on loan", etc.), and not all collections want information at this level of detail to be publicly visible.
